### DISCUSSION TITLE
네이버 매크로 창을 한 계정으로 여러개

### DISCUSSION THREAD

**Discussion by k-spoon** [kimytsc/covid-rest-vaccine-macro#83] [Q&A]
네이버 매크로 창을 한 계정으로 두개정도 띄어놓으면 락걸릴까요? 아니면 두개의 계정으로 각각 띄어놓으면 그건 문제 없을까요?

**Comment by kimytsc** [comment]
락 걸리는 조건은 저도 모르기에 한 계정에 한 개 정도를 추천해 드리며, 두 계정으로 돌리는 것도 가능하다 정도로만 답변드리겠습니다.
